### PR TITLE
[dhcp_relay] Add DHCPv6 Relay Counter test

### PR DESCRIPTION
**Pull request by kellyyeh**
### Description of PR

Summary: Test for DHCPv6 Relay Counter
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 201911

#### What is the motivation for this PR?
Test if the DHCPv6 Relay Counter is working properly.

#### How did you do it?
Run the test after test_dhcp_relay_default, where if the test succeeded, counter would've incremented for the following messages: "Solicit", "Advertise", "Request", "Reply", "Relay-Forward", "Relay-Reply" 
Check if the count is greater than 0.

Counters would not increment if dhcp6relay is not working properly, and this test would automatically fail if test_dhcp_relay_default fails.

#### How did you verify/test it?
Run on vs testbed

### CODE REVIEW

File: tests/dhcp_relay/test_dhcpv6_relay.py [hunk 1]
```diff
@@ -129,6 +129,18 @@ def test_dhcp_relay_default(ptfhost, duthosts, rand_one_dut_hostname, dut_dhcp_r
...
+    """ Test DHCPv6 Counter """
+    duthost = duthosts[rand_one_dut_hostname]
+
+    messages = ["Solicit", "Advertise", "Request", "Reply", "Relay-Forward", "Relay-Reply"]
+
+    for dhcp_relay in dut_dhcp_relay_data:
+        for message in messages:
+            get_message = 'sonic-db-cli STATE_DB hget "DHCPv6_COUNTER_TABLE|{}" {}'.format(dhcp_relay['downlink_vlan_iface']['name'], message)
+            message_count = duthost.shell(get_message)['stdout']
+            assert int(message_count) > 0, "Missing {} count".format(message)
```
Comment (yxieca) [inline tests/dhcp_relay/test_dhcpv6_relay.py:142]: Will this cause false positive if the count is 0?

**Comment by yxieca** [comment]
Can you look into the LGTM alert?

File: tests/dhcp_relay/test_dhcpv6_relay.py (hunk 1 above)
Comment (kellyyeh) [reply tests/dhcp_relay/test_dhcpv6_relay.py:142]: The test fails if count is 0

**Comment by kellyyeh** [comment]
> Can you look into the LGTM alert?

The outdated LGTM alert had already been fixed.

**Comment by kellyyeh** [comment]
@yxieca Can you please review the updated changes? Thanks

**Review by yxieca — APPROVED** [review]

**Pull request merged**